### PR TITLE
REDIRECT_SIGNOUT - Updated

### PR DESCRIPTION
**Pull request by keeysnc**
### Description
*Cleaned up branch for 2671*
https://qmacbis.atlassian.net/jira/software/c/projects/MDCT/boards/232?modal=detail&selectedIssue=MDCT-2672

Adding redirects for cognito redirect sign out based on environment.

prod: https://idm.cms.gov/app/UserHome
val: https://impl.home.idm.cms.gov/app/UserHome 
test: https://test.home.idm.cms.gov/app/UserHome 

### Related ticket(s)

https://qmacbis.atlassian.net/browse/MDCT-2638
MDCT-2638

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by BearHanded — APPROVED** [review]

**Review by karla-vm — APPROVED** [review]

**Review by cassandradanger — DISMISSED** [review]
looks good to me!

**Review by BearHanded — CHANGES_REQUESTED** [review]
Please add the same changes we needed in this change set to prevent the IDM integration from breaking

**Review by karla-vm — APPROVED** [review]
LGTM

File: services/ui-src/.env_example [hunk 1]
```diff
@@ -4,7 +4,7 @@ APPLICATION_ENDPOINT=https://mdctmcrdev.cms.gov/
 COGNITO_IDENTITY_POOL_ID=us-east-1:76708bb0-a458-4ea7-b90e-995ff5da5ab6
 COGNITO_IDP_NAME=OktaOIDC
 COGNITO_REDIRECT_SIGNIN=http://localhost:3000/
-COGNITO_REDIRECT_SIGNOUT=http://localhost:3000/
```
Comment (BearHanded) [inline services/ui-src/.env_example:7]: I don't think `COGNITO_REDIRECT_SIGNOUT=http://localhost:3000/` needs to be removed, just adding the new one should do the trick

**Review by BearHanded — CHANGES_REQUESTED** [review]

File: services/ui-src/.env_example (hunk 1 above)
Comment (keeysnc) [reply services/ui-src/.env_example:7]: Thanks, @BearHanded. It should be updated now.

**Review by BearHanded — APPROVED** [review]
Looks good! Once this is in dev, let's just double check it works and that IDM still works

**Pull request merged**